core/src/language/python.rs

```diff
@@ -636,69 +636,24 @@ impl Python {
                     ty,
                     shared: variant_shared,
                 } => {
-                    variant_class_names.push(variant_class_name.clone());
+                    let tuple_name = self
+                        .format_type(ty, shared.generic_types.as_slice())
+                        .unwrap();
```

Comment:
Could we handle the error without having to `unwrap` or is that how its all done throughout the other languages when they call this fcn?

Reply:
Fixed